.github/workflows/commitlint.yml

```
@@ -3,32 +3,35 @@ name: commitlint
-        if: github.event_name == 'pull_request'
         env:
           BASE_REF: ${{ github.event.pull_request.base.ref }}
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
         run: |
           git fetch origin "$BASE_REF"
           FROM=$(git merge-base "origin/$BASE_REF" "$HEAD_SHA")
           npx commitlint --from "$FROM" --to "$HEAD_SHA" --verbose
+
       - name: Validate PR title with commitlint
-        if: github.event_name == 'pull_request'
         run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)